app/mailers/application_mailer.rb

```diff
@@ -15,7 +15,7 @@ def mail(headers = {}, &block)
 
   class GroupMail
     def initialize(emails)
-      ActiveSupport::Deprecation.warn 'GroupMail will be removed as mailers should not generate multiple messages, use MailNotification#deliver'
+      Foreman::Deprecation.deprecation_warning("1.10.0", controller: "MailNotification", method: "deliver", info: "GroupMail will be removed as mailers should not generate multiple messages")
```

Comment:
@eLobato has suggested we keep to hash rockets style until the codebase is changed as a whole, so `=>` might be preferable for now.